src/components/UnknownAccountWarning.js

```diff
@@ -21,19 +21,31 @@ import { StyleSheet, Text, View } from 'react-native';
 import colors from '../colors';
 import fonts from '../fonts';
 
-export default function UnknownAccountWarning() {
+export default function UnknownAccountWarning({ isPath }) {
 	return (
 		<View style={styles.warningView}>
 			<Text style={styles.warningTitle}>Warning</Text>
-			<Text style={styles.warningText}>
-				This account wasn't retrieved successfully. This could be because its
-				network isn't supported, or you upgraded Parity Signer without wiping
-				your device and this account couldn't be migrated.
-				{'\n'}
-				{'\n'}
-				To be able to use this account you need to:{'\n'}- write down its
-				recovery phrase{'\n'}- delete it{'\n'}- recover/derive it{'\n'}
-			</Text>
+			{isPath ? (
+				<Text style={styles.warningText}>
+					The account is now using Polkadot Canary Address Format.
+					{'\n'}
+					In the future user may be able specify different address formats like
+					Polkadot Live, which may lead to a different address.
```

Comment:
This will not age well, "in the future" doesn't tell much to a user that will not want to update.
I'd go for explaining the current situation that's it. Also Canary network: Kusama.
suggestion:
> This account is not bond to a specific network. The address currently displayed is using Kusama format, however, this account can be used for any network. The address may be different.

Reply:
> however, this account can be used for any network. The address may be different.

`used` maybe misleading, so I suggest rephrase it to.

> however, this account can be encoded by another format, with a different address.